components/admin/Textinput.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
```

Comment:
name of the component is not in camelcase and start with an uppercase

Reply:
Now that I see this I see that some of the components you made starts with an uppercase when it should always start with lowercase. Could you change that everywhere ? Thx!